{
  "Header": {
    "Original": "CVE-2021-20213 (A flaw was found in Privoxy in versions before 3.0.29. Dereference of  ...)",
    "Line": 183027,
    "ID": "CVE-2021-20213",
    "Description": "(A flaw was found in Privoxy in versions before 3.0.29. Dereference of  ...)"
  },
  "Annotations": [
    {
      "Original": "{DLA-2548-1}",
      "Line": 183028,
      "Type": "xref",
      "Bugs": [
        "DLA-2548-1"
      ]
    },
    {
      "Original": "- privoxy 3.0.29-1",
      "Line": 183029,
      "Type": "package",
      "Package": "privoxy",
      "Kind": "fixed",
      "Version": "3.0.29-1"
    },
    {
      "Original": "[buster] - privoxy 3.0.28-2+deb10u1",
      "Line": 183030,
      "Type": "package",
      "Release": "buster",
      "Package": "privoxy",
      "Kind": "fixed",
      "Version": "3.0.28-2+deb10u1"
    },
    {
      "Original": "NOTE: https://www.openwall.com/lists/oss-security/2021/02/03/3",
      "Line": 183031,
      "Type": "NOTE",
      "Description": "https://www.openwall.com/lists/oss-security/2021/02/03/3"
    },
    {
      "Original": "NOTE: https://www.privoxy.org/gitweb/?p=privoxy.git;a=commit;h=75301323495579ff27bdaaea67e31e2df83475fc (3.0.29)",
      "Line": 183032,
      "Type": "NOTE",
      "Description": "https://www.privoxy.org/gitweb/?p=privoxy.git;a=commit;h=75301323495579ff27bdaaea67e31e2df83475fc (3.0.29)"
    }
  ]
}